docs/guide/concepts.md

```diff
@@ -16,48 +16,48 @@ sidebarDepth: 2
 
 `type State = any`
 
-State 表示 Model 的状态数据，通常表现为一个 javascript 对象（当然它可以是任何值）；操作的时候每次都要当作不可变数据（immutable data）来对待，保证每次都是全新对象，没有引用关系，这样才能保证 State 的独立性，便于测试和追踪变化。
+State represents the state data of the Model, usually represented as a javascript object (of course it can be any type). It is treated as immutable data, ensuring that each time it is a new object. No Quote relationships so that State independence is guaranteed, making it easy to test and track changes.
 
-在 dva 中你可以通过 dva 的实例属性 `_store` 看到顶部的 state 数据，但是通常你很少会用到:
+In dva you can see the top state data via the dva instance property `_store`, but usually you will rarely use it:
```

Comment:
In dva you can access the top state data via the dva instance property `_store`, but ~~usually~~ you will rarely use it: